content/docs/introduction/roadmap.md

```diff
@@ -48,6 +48,23 @@ utilized in the Prometheus server. We plan on making use of this
 metadata in the future. The first step is to aggregate this data in-memory
 in Prometheus and provide it via an experimental API endpoint.
 
+### First-class metric type support
+
+Prometheus breaks up notions of first class metric types such as Histograms or Summaries
+into a flattened time series based representation in various places. Prometheus' database
+and query language are both time-series oriented. The exposition formats provide different
+degrees of first-class metrics.
+  
+The protocol buffer exposition format provides full first-class metrics. The text format
```

Comment:
The text format also provides first class metrics, it's how Prometheus interprets them and the general data model that does the flattening.

The text and proto formats are (mostly) equivalent, and proto is a distraction from the actual issues. 

I'd remove this sentence.